fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1759,6 +1759,15 @@ struct ConfigureTenantImpl {
 		     ++configItr) {
 			if (configItr->first == "tenant_group"_sr) {
 				wait(updateTenantGroup(self, tr, self->updatedEntry, configItr->second));
+			} else if (configItr->first == "assigned_cluster"_sr &&
+			           configItr->second != tenantEntry.get().assignedCluster) {
+				auto& newClusterName = configItr->second;
+				TraceEvent("UpdateManagementCluster")
```

Comment:
```suggestion
				TraceEvent(SevWarn, "CannotChangeAssignedCluster")
```

Reply:
It might also be helpful to include the tenant name as a detail

Reply:
Can we also include the above suggestion to change the trace event type name and severity?